blog/2021-05-06-ThePathToOptaPlannerEnlightenmentStartsInTheLog.adoc

```diff
@@ -0,0 +1,311 @@
+= The path to OptaPlanner enlightenment starts in the logs
+:page-interpolate: true
+:awestruct-author: ge0ffrey
+:awestruct-layout: blogPostBase
+:awestruct-tags: [algorithm, insight, production]
+:awestruct-share_image_filename: generalPhaseSequence.png
+
+Do you want to understand what OptaPlanner is doing while it is solving?
+Which decisions it makes? When? And why?
+Do you want to open the box and take a look inside?
+If so, keep reading.
+
+_The Age of Enlightenment centered on the use of reason and the evidence of the senses.
+Similarly, OptaPlanner enlightenment starts with the use of reason and the evidence in the logs._
+
+OK. That's enough philosophy for this article... It's time to open the box.
+Take a look at the log for https://youtu.be/7IuOA9n6kh0[the school timetabling AI quickstart]:
+
+[source]
+----
```

Comment:
Preferably, use a [literal block](https://docs.asciidoctor.org/asciidoc/latest/verbatim/literal-blocks/#delimited-literal-block) for log outputs. [Source](https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/) is for syntax highlighting.

The consequence of using `[source]` for log output in this PR is that some blocks are rendered without syntax highlighting and some blocks are detected as CSS code and are highlighted in a funny way.

Reply:
This is exactly what I was looking for! Thank you!